src/rgw/setup/rgw_service

```diff
@@ -37,9 +37,13 @@ while [ $# -gt 0 ];  do
     shift 1
 done
 
+log_dir="$(dirname "${log_file}")"
```

Comment:
what is log_dir, motr_log_path is not set.
rgw will not even start because we are redirecting the output to log_file
check for variable if non zero and errorout if empty.
for ex: [ -z "log_dir" ] && echo "ERROR: log directory is not set" && exit 0

Reply:
All these args is handled in rgw.py script.
Also, after latest changes replaced this shell script file with python script